cron/monthly/host-settlement.js

```diff
@@ -320,7 +320,7 @@ export async function run() {
     const transactions = hostTransactions.map(t => {
       if (t.source === 'Shared Revenue') {
         // In this context, the original t.amount is actually -t.hostFeeInHostCurrency
-        t.amount = round(t.amount * ((t.data?.hostFeeSharePercent || hostFeeSharePercent) / 100));
+        t.amount = t.amount * ((t.data?.hostFeeSharePercent || hostFeeSharePercent) / 100);
```

Comment:
If amount=333 and percentage=0.1, we'll get `33.300000000000004` here. We can eventually do `round(x, 2)`

```suggestion
        t.amount = round(t.amount * ((t.data?.hostFeeSharePercent || hostFeeSharePercent) / 100));
```

Reply:
This change is a bit out of context but I had problems re-conciliating some amounts and the rounding was ultimately the cause. This should ultimately be scrapped when we implement `HOST_FEE_SHARE`.

But something to keep in mind, when the `HOST_FEE_SHARE` is taken from Stripe with `application_fee`, it's rounded there and this is this precise value that we need to report.

Reply:
Maybe we can avoid the previous rounding issue by setting a precision?

```suggestion
        t.amount = round(t.amount * ((t.data?.hostFeeSharePercent || hostFeeSharePercent) / 100), 2);
```